server/i18n/en.json

```diff
@@ -8009,7 +8009,7 @@
   },
   {
     "id": "interactive_message.decode_trigger_id.expired",
-    "translation": "Trigger ID for interactive dialog is expired. Trigger IDs live for a maximum of {{.Seconds}} seconds."
+    "translation": "Trigger ID for interactive dialog is expired. Trigger IDs live for a maximum of {{.Duration}}."
```

Comment:
Is `Duration` translation friendly?

Reply:
I don't quite get what you are referring to. `Duration` does not need to get translated. The translator only needs to put the variable in the right place.

Reply:
Thanks, that's what I was asking.